### DISCUSSION TITLE
Favorites import and POI

### DISCUSSION THREAD

**Discussion by hjonin** [osmandapp/OsmAnd#16357] [Q&A] [answered]
Hi!

I am migrating from another app to OsmAnd. I would like to import favorites. I checked the gpx format for import and I can see that a favorite item (`<wpt>`) does not seem to be linked to any existing POI (no identifier).
I was wondering, is there a way to import and dynamically link these favorites ? Indeed, I have the feeling they won't be able to have their information (like opening hours) updated if they stay static as they seem to be.

Thanks in advance for your answers!

Helene

**Comment by vshcherb** [comment, marked as answer]
Favorites are linked to POI by name and location (sometimes type)

**Reply by hjonin** [reply]
Thanks for your answer!
